pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -598,7 +599,7 @@ func newHTTPPassThroughFilterChain(configgen *ConfigGeneratorImpl,
 	return filterChains
 }
 
-func buildFallthroughNetworkFilters(push *model.PushContext, node *model.Proxy) []*listener.Filter {
+func buildOutboundCatchAllNetworkFiltersOnly(push *model.PushContext, node *model.Proxy) []*listener.Filter {
```

Comment:
depending on what we decide on the virtual hosts method name, should this also be called `buildOutboundAllowAnyOrBlockAllNetworkFilters()` or if we decide to call the virtual host function as `buildCatchAllVirtualHost`, this is seems to be consistent with that name.